lms/djangoapps/courseware/module_render.py

```diff
@@ -182,6 +183,7 @@ def toc_for_course(request, course, active_chapter, active_section, field_data_c
                                      })
             toc_chapters.append({
                 'display_name': chapter.display_name_with_default,
+                'display_id': name_without_spaces,
```

Comment:
I personally like the default (blue glow) focus state. It's being overridden somewhere in the platform to be dotted lines, which is why it's showing up here funky. I'm thinking maybe we remove the gray background on `focus` which would remove the confusion. Thoughts?